src/api/app/models/project.rb

```diff
@@ -1253,17 +1253,17 @@ def incoming_requests
 
   def outgoing_requests
     incidents = BsRequest.with_involved_projects(id)
-    .or(BsRequest.from_source_project(name))
-    .in_states(:new)
-    .with_types(:maintenance_incident)
-    .pluck(:number)
+                         .or(BsRequest.from_source_project(name))
+                         .in_states(:new)
+                         .with_types(:maintenance_incident)
+                         .pluck(:number)
 
     if is_maintenance?
       maintenance_release = BsRequest.with_target_subprojects(name + ':%')
-                      .or(BsRequest.with_source_subprojects(name + ':%'))
-                      .in_states(:new)
-                      .with_types(:maintenance_release)
-                      .pluck(:number)
+                                     .or(BsRequest.with_source_subprojects(name + ':%'))
+                                     .in_states(:new)
+                                     .with_types(:maintenance_release)
+                                     .pluck(:number)
```

Comment:
@jesus-montano try to run locally ```rake dev:lint:rubocop:auto_correct```  to fix those identations.. then please squash the commits related with that

Reply:
oh, great ty I will do it